mantle-1/upgrades/v1.0.0_upgrade.md

```diff
@@ -0,0 +1,42 @@
+# v1.0.0 Upgrade
+
+The upgrade is scheduled for block `7135001`, August 18, 2023 at UTC 12:00.
+
+### Go version
+
+We recommend using go version 1.19
+
+### Steps for upgrade
+
+Verify you are currently running the correct version (v0.3.1) of the mantleNode
+
+```bash
+$ mantleNode version --long
+name: AssetMantle
+server_name: mantleNode
+version: HEAD-5b2b0dcb37b107b0e0c1eaf9e907aa9f1a1992d9
+commit: 5b2b0dcb37b107b0e0c1eaf9e907aa9f1a1992d9
```

Comment:
commit hash will change after final tag

Reply:
It's for verifying current running version of mantleNode